src/dialogs/notifications/notification-drawer.js

```diff
@@ -63,7 +63,7 @@ export class HuiNotificationDrawer extends EventsMixin(
     <app-drawer id="drawer" opened="{{open}}" disable-swipe align="start">
       <app-toolbar>
         <div main-title>[[localize('ui.notification_drawer.title')]]</div>
-        <ha-icon-button-prev on-click="_closeDrawer" aria-label$="[[localize('ui.notification_drawer.close')]]"></ha-icon-button-prev>
+        <ha-icon-button-prev on-click="_closeDrawer" label$="[[localize('ui.notification_drawer.close')]]"></ha-icon-button-prev>
```

Comment:
`ha-icon-button-prev` would not do anything with an attribute label?

Reply:
True, since I had only covered the "-arrow-" versions, not the ones using the chevron icon. Will be fixed with my next commit.